### DISCUSSION TITLE
Grupo N_2 - My PyC - Tarea T1 -  CT1

### DISCUSSION THREAD

**Discussion by JonTerrex** [santiemanuel/milProgramadoresQA#28] [Simulacro Trabajo Grupal]
class Gabinete():

Gabinete. Tiene que tener capacidad en espacio y refrigeración.

Placa madre. Marca de micro. Tipo de ram. Cantidad SATA o M2 para el almacenamiento. Tamaño del gabinete itx, o atx compatible. No dice pcie.

CPU. Marca, arquitectura para la mother. Tipo de memoria ram. Temperatura que disipe el gabinete. Watts de potencia de la fuente.
Memoria Ram. Tipo compatible con micro y mother.

Almacenamiento. M2 o SATA. Con motherboard.

Placa video. Temperaturas adecuadas al gabinete. Cable PCIe de la fuente. Tamaño para el gabinete. El mother no dice nada de los pcie, pero también habir aque ponerlo. Tampoco dice de la potencia de la fuente y es importante.

Fuente. Potencia para alimentar todo. Gabinete(coolers) cpu, cantidad discos, placas videos. Cantidad de cables pcie para placas videos, pero no dice cables para alimentar almacenamiento SATA

**Comment by arielfherrera** [comment]
En la Placa madre hay que tener en cuenta el tipo de micro AMD o Intel y a la vez la arquitectura de cada una.
La memoria el tipo puede ser ddr2, 3 o 4.

**Reply by ailednitram** [reply]
Ya hay ddr 5 no? O vi mal?

**Reply by santiemanuel** [reply]
Sí, DDR5 salió en las últimas generaciones de AMD y de Intel.

**Comment by ailednitram** [comment]
![2148CA35-D55F-453C-B4BB-D0BFFF92F939](https://user-images.githubusercontent.com/95146193/198705217-d636ed5a-cd4a-4cf5-932e-4f178370f8ae.jpeg)

**Reply by santiemanuel** [reply]
Buenas, ahora te hago una lista de cosas que podrían agregarse a la lista:

- La mother a veces trae un conector de alimentación extra para la CPU. Entonces la fuente necesitaría tener la cantidad suficiente para conectarlos. Esto se usa si la CPU es de alto rendimiento o se quiere hacer overclock
- Entre la CPU y la RAM hay una limitación especial. En la velocidad, si la CPU no soporta la velocidad máxima de la RAM, la RAM funcionará a la máxima velocidad soportada por la CPU. Este caso es especial porque sí funcionará aunque a menor rendimiento
- Lo mismo pasaría entre la mother y la GPU, supongamos que la mother soporta PCIe 3.0 y la placa de video 5.0, entonces la mother limitará el rendimiento máximo de la GPU, pero lo mismo funcionará
- En almacenamiento la versión de SATA no es relevante porque ya todas las mother desde 2009 traen la 3.0 que es la máxima velocidad y es compatible para discos rígidos como sólidos.

Eso sería todo, si tienen alguna duda me consultan en el hilo de esta respuesta para que avancen a la siguiente tarea.

**Reply by ailednitram** [reply]
Ok, Gracias Santi. Ahí va de nuevo con las correcciones.

**Reply by ailednitram** [reply]
![TP Grupal Python - Tarea 1](https://user-images.githubusercontent.com/95146193/198898416-e5cbf88d-ac9d-405b-836f-5cba2db0267c.jpg)

**Reply by santiemanuel** [reply]
Ahí ya quedó bien. Mañana ya publicaré la segunda etapa para que avancen, aunque ya lunes se presentan los temas oficiales. Pero como todavía falta contenido pueden trabajar en los requisitos del que elijan, y luego si quieren seguir practicando en grupo continuar con este proyecto opcional.
